### ISSUE TITLE
Articles with video content should have their listing images with 'video' icon watermark

### ISSUE THREAD

**Issue by xaralis** [pirati-web/jekyll-theme-pirati#10]
This allows people visiting the site to see what articles contain video right away.

It would also be nice if the video could take place of main image for this type of content.

**Comment by xaralis** [comment]
Now possible, when post gets `youtubeVideoId` property within it's front matter, such as:

```
---
date: 2018-06-11
category: blog
tags: tag1 tag2
layout: post
title: Článek s videem místo obrázku
author: Jmeno Prijmeni
authorId: jakub.pirat    # uid nekoho z _people (nepoviné)
image: posts/obrazek.u.tohoto.postu.jpg
youtubeVideoId: 5f0-nQLHHM4 # if youtubeVideoId is provided, it will replace the image as the main content
---
```

Implemented in 2d4b8a4

**Comment by xaralis** [comment]
![image](https://user-images.githubusercontent.com/441739/41927164-b9d4301c-7971-11e8-8538-e23927d53c01.png)
![image](https://user-images.githubusercontent.com/441739/41927177-c47b400a-7971-11e8-9b1a-363d60953308.png)
